src/Generators/AbstractTagGenerator.php

```diff
@@ -168,14 +154,32 @@ public function getExistingTagCommentByTagString($tagString)
     {
         foreach ($this->getExistingTags() as $tag) {
             $content = $tag->getContent();
-            if (strpos($content, $tagString) !== false) {
+            // A tag should be followed by a space before it's description
+            // This is to prevent `TestThing` and `Test` to be seen as the same, when the shorter
+            // is after the longer name
+            if (strpos($content, $tagString . ' ') !== false) {
                 return str_replace($tagString, '', $content);
             }
         }
 
         return '';
     }
 
+    /**
+     * Check if we need to use the short name for a class
+     *
+     * @param string $class
+     * @return string
+     */
+    protected function getAnnotationClassName($class)
+    {
+        if (Config::inst()->get(DataObjectAnnotator::class, 'use_short_name')) {
```

Comment:
You can just `DataObjectAnnotator::config()->get('use_short_name');` now - although that may just be preference but you do save 9 characters each time! :)

Reply:
Good point :) Yeps, I'll need to adjust that :)